### PR TITLE
[refactor] header nav js

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
WIP
- Refactor nav js to can use in doppler webapp
- Clean unuse js

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -1,57 +1,27 @@
...
-    $headerMain.addClass('header-open');
-    $(this).find('.sub-menu').addClass('open');
-  });
-
-  $('.menu-main li a').click(function() {
-    $('.menu-main li a').removeClass('active');
-    $(this).addClass('active');
   });
 
   $('#user-menu--open').click(function() {
```
Comment (andresmoschini) [inline assets/js/main.js]: I think that it is not a good idea defining event handlers on _mouseMove_ on _body_.

Did you consider using _on_ jquery method?

Comment (fcoronelmakingsense) [reply assets/js/main.js]: on what? need a event on body. What event do you thinks works likes mouseMove ?

**Comment by fcoronelmakingsense** [comment]
update pr using "on" event

File: assets/js/main.js [hunk 2]
```diff
@@ -1,57 +1,29 @@
 $(document).ready(function() {
```
Comment (andresmoschini) [inline assets/js/main.js]: Maybe ready is not necessary anymore?

Comment (fcoronelmakingsense) [reply assets/js/main.js]: yes, i removed now and works

**Comment by andresmoschini** [comment]
I love it! great work @fcoronelmakingsense !

**Comment by cbernat** [comment]
Very nice!

**Pull request merged**